ci-operator/step-registry/rosa/e2e/test/rosa-e2e-test-commands.sh

```
@@ -61,19 +61,20 @@ fi
 
 # Run tests
 GINKGO_FLAGS="--ginkgo.junit-report=${ARTIFACT_DIR}/junit-rosa-e2e.xml --ginkgo.v"
-if [[ -n "${LABEL_FILTER}" ]]; then
-  GINKGO_FLAGS="${GINKGO_FLAGS} --ginkgo.label-filter=${LABEL_FILTER}"
+log "LABEL_FILTER='${LABEL_FILTER:-}'"
+if [[ -n "${LABEL_FILTER:-}" ]]; then
+  GINKGO_FLAGS="${GINKGO_FLAGS} --ginkgo.label-filter '${LABEL_FILTER}'"
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)